### ISSUE TITLE
Unconfirmed Transaction

### ISSUE THREAD

**Issue by Themanofsteel3** [keepkey/multibit-hd#930]
Hey,

A few days ago i transferred 3.79 BTC to my wallet, the funds went through and confirmed all the way and i even confirmed that by looking at it on the Blockchain. The next day the balance turned to unconfirmed transaction. Now usually when ever this happens i repair my wallet and the funds are back, This time it did not. I repaired my wallet countless times and still no results, At this point i have given up all hope on retrieving my 3.79 BTC and i have moved to a different wallet software. Which is truly a disappointment because it is hard to find a software as streamlined as Multibig HD. Anyway that being the case i hope that anyone here can help me.

Before posting this thread i looked into all the previous posts that went through this same issue and all of the trouble shooting steps mentioned in the solutions to those problems including Uninstalling the software and restoring with wallet words, i even renamed the back up folder to 'old-mbhd-<the long number>' and still i have gotten no results so at this point i have completely given up and this really lowers my faith in the BTC system because of this whole issue, i really hope someone out there can help me.

Here is the block chain link for the transactions , You can see the unspent outputs all equal out to 3.79 BTC:

https://blockchain.info/address/16C3Ac8uYS4gMLrVPzCq8v9s5MrgzM7aHe

Here is a screen shot of my wallet so you can clearly see the 3.79 unconfirmed funds:

![screen shot 2016-10-01 at 6 29 44 pm](https://cloud.githubusercontent.com/assets/22567343/19017379/3aae4372-8805-11e6-9ee6-08a2aacf16ff.png)

I hope anyone can help

Thank you

**Comment by kevinm45** [comment]
Hello,

I am having a similar issue, my balance became unconfirmed today after sending some bitcoins. I had sent bitcoins earlier as well and it was fine. The I did a second transaction and the balance just went to unconfirmed and all past transactions were unconfirmed. 

I have repaired the wallet several times as well, after the 3rd or 4th repair the past transactions became confirmed, except the last one. Balance is still unconfirmed.

I followed https://bitcointalk.org/index.php?topic=1361928.msg13927284#msg13927284 and renamed the wallet directory folder, and restored my wallet on my computer, still no luck however. I even tried loading and restorting MultiBit on another computer but no fix.

Please let me know if there is another fix for this. 

I even tried emptying my wallet to another wallet, but I can't since its unconfirmed.

Hope someone can help

**Comment by Themanofsteel3** [comment]
Hey,

My BTC are still unconfirmed, i also tried all the steps that were given in that thread and also everything you mentioned above, with no luck still.

I hope someone out there can help.

Kind regards.

**Comment by kevinm45** [comment]
I don't know whats going on.

I made a new wallet, and put bitcoins in there as well. The bitcoins were confirmed, and I was able to make transactions, but the transactions haven't confirmed. I tried repairing the new wallet, but my remaining balance became unconfirmed.

The bitcoins I got are marked as Received in the Payments Transactions, and the bitcoins I sent is still at Sending with the two arrows, but my balance is completely Unconfirmed.

Multibit is not doing to well for me today.
